src/cpp/Canestra/src/capteur.cpp

```diff
@@ -414,8 +414,8 @@ void Camera::calc_visi(Liste <Diffuseur*>& Ldiff)
 void Camera::maj_stat()
  { double x;
 
-   for(register int i=0;i<img->taille(0);i++)
-     for(register int j=0;j<img->taille(1);j++)
+   for(int i=0;i<img->taille(0);i++)
+     for(int j=0;j<img->taille(1);j++)
          { x=img->val(i,j);
```

Comment:
int i=0, j=0;
outside the loop

Reply:
What is the most effective way to do that ?

Reply:
Just do 
```C
int i=0, j=0;
for(i=0;i<img->taille(0);i++)
     for( j=0;j<img->taille(1);j++)
         { x=img->val(i,j);
```